packages/bridge/src/runtime/composables/asyncData.ts

```diff
@@ -142,7 +142,7 @@ export function useAsyncData<
     nuxt._asyncData[key] = {
       data: _ref(options.getCachedData!(key) ?? options.default!()),
       pending: ref(!hasCachedData()),
-      error: toRef(nuxt.payload._errors, key),
+      error: ref(nuxt.payload._errors[key]),
```

Comment:
I believe this will mean that the error is 'disconnected' from the payload and won't be included in it.

Is there a reason that `toRef` isn't reactive here? If so, we might need to add an additional assignment to the payload when the error is thrown below.

Reply:
Ah, it seems this wasn't essential. It might be because `_errors` is not reactive.
https://github.com/nuxt/bridge/blob/8244bdaf91d04568fa5d8c758873a82a63deb912/packages/bridge/src/runtime/app.plugin.mjs#L9

Reply:
Yes, that might be right 👍